moped-database/views/project_list_view.sql

```diff
@@ -82,6 +82,7 @@ AS WITH project_person_list_lookup AS (
       WHERE true
         AND phases.project_id = mp.project_id 
         AND phases.phase_name = 'construction'::text
+        AND phases.is_deleted = false
```

Comment:
These two updates to the view are just to make sure that construction and complete phase dates don't stick around after a soft delete.